drakrun/drakrun/playground.py

```diff
@@ -87,10 +87,25 @@ def drakvuf(self, plugins, timeout=60):
 
         return d
 
+    def help(self):
+        usage = """
+        Available commands:
+        - copy(file_path)   # copy file onto vm desktop
+        - mount(iso_path)   # mount iso, useful for installing software, e.g. office
+        - drakvuf(plugins)  # start drakvuf with provided set of plugins
+        - run(cmd)          # run command inside vm
+        - exit()            # exit playground, remember to run postinstall if you want to save vm state
+        """
```

Comment:
I think we should apply https://docs.python.org/3/library/textwrap.html#textwrap.dedent

Reply:
Cool, thanks for the link